src/graphql/schema.graphql

```diff
@@ -5,10 +5,16 @@ type Query {
   contributorInformation(
     "The contributor’s GitHub username."
     githubUsername: String!
-  ): ContributorInformation!
+  ): ContributorInformation! @deprecated(reason: "use getContributor instead")
+
+  getContributor(githubUsername: ID!): Contributor
```

Comment:
Continuing from our discussion at the end of the livestream today: if we made all the same calls as `getContributor` _and_ updated tags, we could create a mutation like this:

```
updateContributorTags(githubUsername: String!): Contributor
```

This would allow us to drop `newTags` from the `Contributor` type (`getContributor` would only return existing tags) and would allow the UI to just call `updateContributorTags` on login to receive the correctly tagged `Contributor` data.

This also plugs a security hole I just realized we created by exposing `addTagsToShopifyCustomer` as part of the public GraphQL API: anyone could make a call to add any tags they want to their own user while logged in. 😱 

By making the call based solely on the GitHub username, we can perform checks to ensure that we only add the correct tags for a given username.

So, after much hand-wringing over this, I think we _need_ to make this change because the other way is actually just terrible and wrong. 😅